premailer/premailer.py

```diff
@@ -138,6 +139,14 @@ def __init__(self, html, base_url=None,
                  remove_unset_properties=True):
         self.html = html
         self.base_url = base_url
+
+        # if base_url is specified, premailer will transform all URLs by
+        # joining them with the base_url. Setting preserve_internal_links to
+        # True will disable this behavior for links to named anchors. Setting
+        # preserve_inline_attachments to True will disable this behavior for
+        # any links with cid: scheme. Setting preserve_all_links to True will
+        # disable this behavior altogether.
```

Comment:
This is a good comment but my mind immediately wonders; Why not just *not* set `base_url`?

Reply:
I had to read the code to answer that. The answer is that if you set `base_url` it is used for two things:

1. Rewrite relative links.
2. Correct relative URLs for loading external style sheets. 

The comment should mention that the `base_url` is used for fetching external style sheets as well.

Reply:
Agreed.